vyper/parser/external_call.py

```diff
@@ -180,8 +180,13 @@ def make_external_call(stmt_expr, context):
             gas=gas,
         )
 
-    elif isinstance(stmt_expr.func.value, vy_ast.Attribute) and stmt_expr.func.value.attr in context.globals:  # noqa: E501
-        contract_name = context.globals[stmt_expr.func.value.attr].typ.unit
+    elif (
+        isinstance(stmt_expr.func.value, vy_ast.Attribute) and
+        stmt_expr.func.value.attr in context.globals
+        and hasattr(context.globals[stmt_expr.func.value.attr].typ, 'name')
```

Comment:
What's this line for?

Reply:
Currently, interfaces are being represented as an `address` with a unit, where the unit is the name of the interface.  This solves the same issue by comparing the contract names without needing to use units.